src/index.js

```diff
@@ -1 +1,42 @@
-console.log('hello!!!!!')
+const { makeGrid } = require('../src/grid/grid');
+
+const form = document.getElementById('color-form');
+form.addEventListener('submit', handleSubmit);
+
+function handleSubmit(event) {
+  clearGridContainer();
+  event.preventDefault();
+  const topLeft = addHash(document.getElementById('top-left').value);
```

Comment:
This might be a good place to make use of the FormData() object. According to caniuse it looks pretty supported. https://caniuse.com/?search=formdata%20api

MDN: https://developer.mozilla.org/en-US/docs/Web/API/FormData specifically the values() call.

Example codepen: https://codepen.io/HKeca/pen/JjEvoQQ?editors=1111

Reply:
Thank you Hunter! Looking into adding this now. That example made it so much easier to understand!